### PR TITLE
toolbar fix

### PR DESCRIPTION
**Pull request by bjedrzejewski**
Fix for the issue #202

### CODE REVIEW

**Comment by owennw** [comment]
I think this breaks the minimise and close buttons?

**Comment by bjedrzejewski** [comment]
Yes it does, I am bit blind, let me fix it properly

File: src/assets/styles/toolbar.less [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ﻿#toolbar {
   height: 50px;
   right: 65px;
-  position: relative;
+  left: 350px;
+  position: absolute;
```
Comment (owennw) [inline src/assets/styles/toolbar.less]: We have `@sidebar-width` for that variable.

File: src/assets/styles/toolbar.less [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ﻿#toolbar {
   height: 50px;
   right: 65px;
```
Comment (owennw) [inline src/assets/styles/toolbar.less]: Similarly there's a hard-coded height for `.sidetab-top` in sidebar.less that should be made a variable and reused here.

**Comment by bjedrzejewski** [comment]
@owennw while noticing it, i fixed the splitter height as well, as Rui said, he needs it at 1px

**Pull request merged**